README.md

```diff
@@ -279,6 +279,24 @@ role.
 Similar to `controller` and `vlan`, the `parent` references the connection profile in
 the ansible role.
 
+#### `type: infiniband`
+
+For the infiniband connection, the following options are supported:
+
+- `p_key`: The infiniband P_Key to use for the device. The default value of `-1` means
+  to use the P_Key at the index 0 and the connection will be created on the physical
+  infiniband fabrics. Otherwise, it is a 16-bit unsigned integer and the ipoib (IP OVER
+  INFINIBAND) connection will be created, the high bit should be set if it is a
+  "full membership" P_Key.
+- `transport_mode`: The ipoib (IP OVER INFINIBAND) connection operation mode. The
```

Comment:
Pleas don't use all caps but "IP over InfiniBand" (or however this is written)

Reply:
Sure.